packages/fxa-auth-server/lib/luaScripts/getSessionTokens.lua

```diff
@@ -0,0 +1,52 @@
+local REDIS_SESSION_TOKEN_PROPERTIES = {
```

Comment:
Can lua scripts be unit tested somehow? Are they tested via integration tests?

Reply:
we can test them with mocha via our redis.js against a real redis server

Reply:
We ought to be able to prove backward compatibility with test cases